.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!-- Before creating an issue please make sure you are using the latest version. -->
+
+<!-- If this is a feature request use this portion of the template and delete the bug portion -->
+**Describe the *feature* request**
+
+
+**What is motivation or use case for changing this?**
+
+
+**Please mention other relevant information such as the browser version, Node.js version, Lighthouse version, and Operating System.**
+
+
+<!-- If this is a bug report use this portion of the template and delete the feature request portion -->
+
+**Describe the *bug* issue**
+
+
+**What is the current behavior?**
+
+
+**What is the expected behavior?**
+
+
+**Provide the steps to reproduce if this is a bug**
```

Comment:
we've already established this is the bug part so let's just say `Provide the steps to reproduce`